test/libraries/InstallDeps/git-deps-url-new.hxml

```diff
@@ -0,0 +1 @@
+-lib flixel:git:https://github.com/geokureli/flixel.git#4.5.0
```

Comment:
It should be possible to set up a url that is just a path to a local directory. There is a util function for setting up a git repository, and there is a `test/libraries` folder with some existing libraries that could be pointed to.

https://github.com/HaxeFoundation/haxelib/blob/5a836287828fdaeb6aa91695a5eb399cee0f6640/test/tests/util/Vcs.hx#L8

https://github.com/HaxeFoundation/haxelib/blob/5a836287828fdaeb6aa91695a5eb399cee0f6640/test/tests/integration/TestVcs.hx#L6-L8

External repositories may change ownership or move (and they rely on the network to clone), so it is more reliable to clone from local repositories in tests, especially the unit tests.